lib/type_class.ex

```diff
@@ -263,32 +263,44 @@ defmodule TypeClass do
         def __force_type_instance__, do: @force_type_instance
       end
 
+      print_warnings = case System.get_env("PRINT_WARNINGS") do
+        nil -> nil
+        "false" -> false
+        _ -> true
+      end
+
       cond do
         unquote(class).__force_type_class__() ->
-          # IO.warn("""
-          # The type class #{unquote(class)} has been forced to bypass \
-          # all property checks for all data types. This is very rarely valid, \
-          # as all type classes should have properties associted with them.
-
-          # For more, please see the TypeClass README:
-          # https://github.com/expede/type_class/blob/master/README.md
-          # """)
+          if print_warnings do
```

Comment:
Would be cool to have this as a configuration option too, just as the FORCE_TYPE_CLASS is.